iOS/FVM/Framework/Factories/FvmFactory.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+public class FvmFactory {
+    public class func create(_ initialConfiguration: String, _ completionAction: ((String) -> Void)?) -> FVMDamagedCarViewController {
```

Comment:
I suppose it would be better if create() will be in controller class. What do you think @matrejek ?

Reply:
But it is, you're commented on outdated file ;)